1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,26 @@
 // Example 1
 let a;
 console.log(a);
-
+// It is undefined because it doesn't have value.
 
 // Example 2
 function sayHello() {
-    let message = "Hello";
+  let message = "Hello";
 }
 
 let hello = sayHello();
 console.log(hello);
-
+// It is undefined because it doesn't return value
 
 // Example 3
 function sayHelloToUser(user) {
-    console.log(`Hello ${user}`);
+  console.log(`Hello ${user}`);
 }
 
 sayHelloToUser();
-
+// It is undefined because it hasn't value for user parameter
```

Comment:
Your overall thinking is correct, as you aware that the user argument is not provided. However, the leading Hello exists and will be printed.